.gitignore

```diff
@@ -1,14 +1,16 @@
-/.bundle
-
-/db/*.sqlite3
-/db/*.sqlite3-journal
-
-/log/*
-/tmp/*
-!/log/.keep
-!/tmp/.keep
-/out
-
-.byebug_history
-
-coverage
+/.bundle
+
+/db/*.sqlite3
+/db/*.sqlite3-journal
+/db/schema.rb
+
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+/out
+
+.byebug_history
+
+coverage
+Gemfile.lock

```

Comment:
`Gemfile.lock` is something that we _do_ want to commit since it's essentially a list of the project's gems and their dependencies. It's generated/used by Bundler when installing the gems in the `Gemfile`